packages/data-table/src/helpers/fetchers.ts

```diff
@@ -0,0 +1,14 @@
+import cacheBustingString from '@cdc/core/helpers/cacheBustingString'
+import fetchRemoteData from '@cdc/core/helpers/fetchRemoteData'
+
```

Comment:
I think these helpers might be a little too much abstraction. I see here that there's catch statements on fetchConfig for example but then I see that you're trying to also catch in the CdcDataTable file itself where they are called.

in CdcDashboard for example we are just doing
`const config = (await (await fetch(configUrl)).json())` I'm not sure what the error state would look like for that, but I haven't heard any complaints yet?